pinot-common/src/main/java/org/apache/pinot/common/request/PinotQuery.java

```diff
@@ -155,34 +155,34 @@ public short getThriftFieldId() {
   public static final java.util.Map<_Fields, org.apache.thrift.meta_data.FieldMetaData> metaDataMap;
   static {
     java.util.Map<_Fields, org.apache.thrift.meta_data.FieldMetaData> tmpMap = new java.util.EnumMap<_Fields, org.apache.thrift.meta_data.FieldMetaData>(_Fields.class);
-    tmpMap.put(_Fields.VERSION, new org.apache.thrift.meta_data.FieldMetaData("version", org.apache.thrift.TFieldRequirementType.OPTIONAL, 
```

Comment:
Need to undo these changes. Might have gotten added earlier when style guideline was not imported I think